src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppEcsFargate/Configurations/VpcConfiguration.cs

```diff
@@ -20,5 +20,22 @@ public class VpcConfiguration
         /// then use an existing VPC by referencing through <see cref="VpcId"/>
         /// </summary>
         public string VpcId { get; set; }
+
+#nullable disable warnings
```

Comment:
Can you add a comment why we disable the warning?

Reply:
I added a comment for all classes that are doing this.